OptimizelySDK/Optimizely.cs

```diff
@@ -438,8 +442,8 @@ private bool ValidateInputs(string datafile, bool skipJsonValidation)
             userAttributes = userAttributes ?? new UserAttributes();
 
             var userContext = CreateUserContextCopy(userId, userAttributes);
-            var variation = DecisionService.GetVariation(experiment, userContext, config)?.
-                ResultObject;
+            var variation = DecisionService.GetVariation(experiment, userContext, config)
+                ?.ResultObject;
```

Comment:
I like the use of the leading period in method chaining- what do you think about leaving the null conditional operator on the previous line, grouping it with its relevant method?

Reply:
🤷‍♂️ you win some, you lose some